src/Calendar/Calendar.tsx

```diff
@@ -1,27 +1,120 @@
 import './Calendar.css';
+import {useState, ChangeEvent} from 'react';
 import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import Toolbar from '@material-ui/core/Toolbar';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+
+export const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
as we work heavily with dates, maybe some date module would be a good idea? You could prevent such lists, and operating with time/dates would be easier with `moment` for example?